hasher-matcher-actioner/hmalib/tests/test_pdq_models.py

```diff
@@ -148,34 +133,36 @@ def test_write_hash_record(self):
 
     def test_query_hash_record(self):
         """
-        Test PipelinePDQHashRecord write table with get_from_content_key query
+        Test PipelineHashRecord write table with get_from_content_key query
         """
 
         record = self.get_example_pdq_hash_record()
         record.write_to_table(self.table)
 
-        query_record = models.PipelinePDQHashRecord.get_from_content_id(
+        query_record = models.PipelineHashRecord.get_from_content_id(
             self.table, TestPDQModels.TEST_CONTENT_ID
-        )
+        )[0]
 
         assert record == query_record
 
-    def test_query_hash_record_by_time(self):
-        """
-        Test PipelinePDQHashRecord write table with get_from_content_key query by time
-        """
-
+    def test_query_recent_hash_records(self):
         record = self.get_example_pdq_hash_record()
 
         record.write_to_table(self.table)
 
-        query_record = models.PipelinePDQHashRecord.get_from_time_range(self.table)[0]
+        query_record = models.PipelineHashRecord.get_recent_items_page(
+            self.table
+        ).items[0]
+
+        record.signal_specific_attributes = {}
+        # While signal_specific_attributes are stored in the table, the index
```

Comment:
Since PK and SK are known, the cost of querying a single hash is 10-15ms. Only put that in the index that is useful for list queries.

At this point, I believe keeping the index small has benefits. I do not see signal specific attributes being useful in list queries yet. Should that change, we can revisit this decision.